### PR TITLE
Add designers to teams

### PR DESCRIPTION
**Pull request by AlicjaSuska**
This PR assigns designers to the teams. Now, it is unclear who works with which team.
- when requesting design work or reviews, it'd be easier to assign designers and mention them in PRs
- designers are members of the teams and it should be expressed in the handbook

@rrhyne @quinnkeast please, review. @quinnkeast I've listed 'code monitoring' next to your name but there is no code monitoring team page in the handbook. Also, I've added myself to the code intel for now as I've been doing some work for this team. I'm open to change it though as I'm already assigned to the web and code insights teams.

### CODE REVIEW

File: handbook/product/index.md [hunk 1]
```diff
@@ -12,9 +12,9 @@ The product team consists of the product managers, product designers, and techni
   - [Joel Kwartler](../../../company/team/index.md#joel-kwartler-he-him) - ([Web](../engineering/web/index.md))
   - [Malo Marrec](../../../company/team/index.md#malo-marrec-he-him) - ([Campaigns](../engineering/campaigns/index.md))
 - [Product design](design/index.md)
-  - [Rob Rhyne](../../../company/team/index.md#rob-rhyne)
-  - [Alicja Suska](../../../company/team/index.md#alicja-suska-she-her)
-  - [Quinn Keast](../../../company/team/index.md#quinn-keast-he-him)
+  - [Rob Rhyne](../../../company/team/index.md#rob-rhyne) - ([Campaigns](../engineering/campaigns/index.md)) and ([Search](../engineering//search/index.md))
```
Comment (umpox) [inline handbook/product/index.md:15]:
```suggestion
  - [Rob Rhyne](../../../company/team/index.md#rob-rhyne) - ([Campaigns](../engineering/campaigns/index.md)) and ([Search](../engineering/search/index.md))
```

File: handbook/product/index.md [hunk 2]
```diff
@@ -12,9 +12,9 @@ The product team consists of the product managers, product designers, and techni
   - [Joel Kwartler](../../../company/team/index.md#joel-kwartler-he-him) - ([Web](../engineering/web/index.md))
   - [Malo Marrec](../../../company/team/index.md#malo-marrec-he-him) - ([Campaigns](../engineering/campaigns/index.md))
 - [Product design](design/index.md)
-  - [Rob Rhyne](../../../company/team/index.md#rob-rhyne)
-  - [Alicja Suska](../../../company/team/index.md#alicja-suska-she-her)
-  - [Quinn Keast](../../../company/team/index.md#quinn-keast-he-him)
+  - [Rob Rhyne](../../../company/team/index.md#rob-rhyne) - ([Campaigns](../engineering/campaigns/index.md)) and ([Search](../engineering/search/index.md))
+  - [Alicja Suska](../../../company/team/index.md#alicja-suska-she-her) - ([Web](../engineering/web/index.md)) and ([Code Insights](../engineering/code-insights/index.md))
+  - [Quinn Keast](../../../company/team/index.md#quinn-keast-he-him) - ([Cloud](../engineering/cloud/index.md)) and Code monitoring
```
Comment (quinnkeast) [inline handbook/product/index.md:17]:
```suggestion
  - [Quinn Keast](../../../company/team/index.md#quinn-keast-he-him) - ([Cloud](../engineering/cloud/index.md)) and [Search (Code monitoring & contexts)](../engineering/search/index.md))
```
For the sake of clarity ("what's the code monitoring team?") I've tweaked this to clarify the Search team, but for specific things.

File: handbook/product/index.md [hunk 3]
```diff
@@ -12,9 +12,9 @@ The product team consists of the product managers, product designers, and techni
   - [Joel Kwartler](../../../company/team/index.md#joel-kwartler-he-him) - ([Web](../engineering/web/index.md))
   - [Malo Marrec](../../../company/team/index.md#malo-marrec-he-him) - ([Campaigns](../engineering/campaigns/index.md))
 - [Product design](design/index.md)
-  - [Rob Rhyne](../../../company/team/index.md#rob-rhyne)
-  - [Alicja Suska](../../../company/team/index.md#alicja-suska-she-her)
-  - [Quinn Keast](../../../company/team/index.md#quinn-keast-he-him)
+  - [Rob Rhyne](../../../company/team/index.md#rob-rhyne) - ([Campaigns](../engineering/campaigns/index.md)) and ([Search](../engineering/search/index.md))
+  - [Alicja Suska](../../../company/team/index.md#alicja-suska-she-her) - ([Web](../engineering/web/index.md)) and ([Code Insights](../engineering/code-insights/index.md))
```
Comment (macraig) [inline handbook/product/index.md:16]: @AlicjaSuska I saw you added yourself to the code intel team (yay! :tada:) should team also be added here?

File: handbook/engineering/search/index.md [hunk 4]
```diff
@@ -53,6 +53,7 @@ Iterations start **every other Monday**.
 ## Members
 
 - We're hiring a [Product Manager](https://jobs.lever.co/sourcegraph/254299f5-f91b-43e2-aa1a-3732963dd296) for this role. [Christina Forney](../../../company/team/index.md#christina-forney-she-her) is involved in the meantime.
+- [Rob Rhyne](../../../company/team/index.md#rob-rhyne) ([Product Designer](../../product/roles/product_designer.md))
```
Comment (limitedmage) [inline handbook/engineering/search/index.md:56]: Should @quinnkeast  be added here too (for code monitoring & search contexts)?

File: handbook/engineering/campaigns/index.md [hunk 5]
```diff
@@ -99,7 +99,7 @@ The private channel is for communications that would be of no interest to someon
 The Campaigns team is the current owner of [src-cli](https://github.com/sourcegraph/src-cli), due to the fact that most of the src-cli work in recent months has been related to Campaigns. We do not expect to be the permanent owners of src-cli; when another team becomes the main contributor, we will transfer ownership to them.
 
 ## Members
-- [Rob Rhyne](../../../company/team/index.md#rob-rhyne) acting Product Manager and Product Designer
+- [Rob Rhyne](../../../company/team/index.md#rob-rhyne) acting Product Manager and [Product Designer](../../product/roles/product_designer.md)
```
Comment (rrhyne) [inline handbook/engineering/campaigns/index.md:102]:
```suggestion
- [Rob Rhyne](../../../company/team/index.md#rob-rhyne) [Product Designer](../../product/roles/product_designer.md)
```

**Review by rrhyne — APPROVED** [review]
LGTM

File: handbook/engineering/search/index.md (hunk 4 above)
Comment (AlicjaSuska) [reply handbook/engineering/search/index.md:56]: @quinnkeast @rrhyne my initial idea was to have 1 designer per team. Should we add both of you here?

Comment (AlicjaSuska) [reply handbook/engineering/search/index.md:56]: @quinnkeast @rrhyne my initial idea was to have 1 designer per team. Should I add both of you to the search team?

File: handbook/product/index.md (hunk 3 above)
Comment (AlicjaSuska) [reply handbook/product/index.md]: Just added, good point, thank you @macraig 🎉

File: handbook/engineering/search/index.md (hunk 4 above)
Comment (quinnkeast) [reply handbook/engineering/search/index.md:56]: For clarity in the short term, it may be useful.

**Pull request merged**